child_compassion/view/child_remove_from_internet.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+	<data>
+        <record id="child_remove_from_internet_view" model="ir.ui.view">
+            <field name="name">compassion.child.remove.from.internet</field>
+			<field name="model">child.remove.from.internet</field>
+            <field name="type">form</field>
+			<field name="arch" type="xml">
+                <form string="Put child on internet" version="7.0">
+                    <footer>

```

Comment:
Add also a paragraph telling it will remove the children and recalling to update the index on typo3.